test/js/samples/window-binding-scroll-store/expected.js

```diff
@@ -0,0 +1,133 @@
+import {
```

Comment:
Is this `js` test testing for anything in particular that the other `runtime` test doesn't cover? In general, we like to avoid having tests for particular js output if possible, because they need to be updated when there are changes to the generated code, and runtime tests continue to just work.

Reply:
to take snapshot of the generated output?
I thought it would be great to have both snapshot test and behaviour test?